tests/unit/test_dto/test_factory/test_dataclass_dto.py

```diff
@@ -31,67 +29,6 @@ def fx_dto_type() -> type[DataclassDTO[Model]]:
     return DataclassDTO[Model]
 
 
-@pytest.mark.skipif(sys.version_info >= (3, 9), reason="generic builtin collection")
```

Comment:
same it is removed but we drop 3.8 only no ?

Reply:
The test will run on 3.8 only, since it's being skipped for 3.9 and above. It's checking the generic builtins backwards compatibility for 3.8 specifically. 

We should probably to these checks like `> (3, 8)`, so it's a bit more clear where the boundary is supposed to be